mark.go

```diff
@@ -0,0 +1,46 @@
+package qt
+
+import (
+	"runtime"
+	"testing"
+)
+
+// A Marker records a name at a given Go source file position.
+type Marker struct {
+	_    struct{} // Prevent
+	Name string
+	File string
+	Line int
+}
+
+// Mark returns a Marker with the given name
```

Comment:
We could expand the docstring with an example of how to use this, as part of table testing for instance, by calling `Mark` there and then `Run` in the actual test.